cmd/utils/utils.go

```diff
@@ -35,44 +34,3 @@ func InitConfig() {
 	viper.SetEnvPrefix(constants.ENV_PREFIX)
 	viper.AutomaticEnv()
 }
-
-// saves the config file with the current config parameters.
-//
-// If the config file does not exist, it creates it.
-//
-// If the config file exists, it creates a backup copy ending with .bak
-// and overwrites the existing config file.
-// TODO: consider using one single utility function to save/update/append files throughout the codebase
-func SaveConfig() error {
```

Comment:
I removed it, because it is unused and afaict redundant to `WriteDefaultConfigFile()`. We can always pull it back if we decide we want to save arbitrary config files, but I don't expect that

Reply:
Well it's a bit different, it writes all the parameters currently in use, which is a great help for debugging. For example if we had used this yesterday we would see that environment equals local